Source/GSXib5KeyedUnarchiver.m

```diff
@@ -3166,6 +3231,7 @@ - (BOOL) containsValueForKey: (NSString *)key
         {
           hasValue  = [currentElement attributeForKey: @"title"] != nil;
           hasValue |= [currentElement attributeForKey: @"image"] != nil;
+          hasValue |= [currentElement attributeForKey: @"string"] != nil;
```

Comment:
Shouldn't this be "url"?

Reply:
No.  Url is an element.  Its value is string.

Reply:
Here is an extract from your own XIB file:
<pathCell key="cell" controlSize="small" selectable="YES" editable="YES" alignment="left" id="hSq-Cw-QmP">
                            <font key="font" metaFont="controlContent" size="11"/>
                            <url key="url" string="file:///usr/"/>
</pathCell>
When looking for NSContents we are on the pathCell element and we should check whether there is an element for the key url. At least that is my understanding of what goes on here. Did you add some NSLog statement to check?

Reply:
Handling this in a separate PR.